03week/ticTacToe.js

```diff
@@ -24,23 +24,51 @@ function printBoard() {
 }
 
 function horizontalWin() {
-  // Your code here
+  //horizontalWin(), checks if there are three X's or O's in [0][0], [0][1], [0][2], etc, indexOf method
+  if (printBoard.indexOf(printBoard) === ((([0][0]) && [0][1] && [0][2]) || ([1][0] && [1][1] && [1][2]) || ([2][0] && [2][1] && [2][2])) {
+    return 'You Win!';
+  } else {
+    const getPrompt;
+  }
 }
 
 function verticalWin() {
-  // Your code here
+  //verticalWin(), checks if there are th ree X's or O'ss in [0][0], [1][0], [2][0]; [0][1], [1][1], [2][1]; [0][2], [1][2], [2][2]; indexOf method
+  if (printBoard.indexOf(printBoard) === ((([0][0]) && [1][0] && [2][0]) || ([0][1] && [1][1] && [2][1]) || ([0][2] && [1][2] && [2][2])) {
+    return 'You Win!';
+  } else {
+    const getPrompt;
+  }
 }
 
 function diagonalWin() {
-  // Your code here
+  //diagonalWin(), checks if there are three X's or O's in [0][0], [1][1], [2][2]; [0][2], [1][1], [2][0]; indexOf method
+  if (printBoard.indexOf(printBoard) === ((([0][0]) && [1][1] && [2][2]) || ([0][2] && [1][1] && [2][0])) {
+    return 'You Win!';
+  } else {
+    const getPrompt;
+  }
 }
 
 function checkForWin() {
-  // Your code here
+  //checkForWin(), check if horizontalWin, verticalWin, diagonalWin have three of the same values, if else statements
+  if (diagonalWin || verticalWin || horizontalWin) {
+    return 'Play again?';
+  }
 }
 
 function ticTacToe(row, column) {
-  // Your code here
+  //ticTacToe(), begin the game with player X
+  print(row, column)
+  //playerXTurn(), set playerTurn to 'X' - place an X in the column/row that player X chooses, splice method on var board
+  checkForWin
+  let playerTurn = 'O';
+  //playerOTurn(), change playerTurn to 'O' - switch to player O, place an O in the column/row that player O chooses, splice method on var board
+  //switch back to player X, run the same code again
+  checkForWin
+  let playerTurn = 'X';
```

Comment:
This would throw and error, you already declared playerTurn. If you want to re-assign, it would be playerTurn=''O"